package.yaml

```diff
@@ -48,6 +48,7 @@ dependencies:
   - vector               >= 0.12 && < 0.13
   - websockets           >= 0.12 && < 0.13
   - wuss                 >= 1.1  && < 1.2
+  - exceptions
```

Comment:
Could you upper and lower bound this package? Also, please sort in alphabetical order.

Reply:
Consider not using an upper bound in this case. It's very unlikely that a new version of the library would break the typeclass exports used here. But it is likely that at some point in the future someone will have a problem building your library due to the upper bound (maybe not because this library isn't really being changed at all anymore, but that case works just fine without the upper bound as well).

The lower bound seems to be 0.4